src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js

```diff
@@ -48,7 +49,10 @@ export default function ApplicationForm({
     ""
   );
 
-  const history = useHistory();
+  const updateInput = (input, setInput, setError) => {
```

Comment:
do you need to pass in `setInput` and `setError` in the function, or can you just access them cause they're within scope?

Reply:
I wasnt sure, since the function is being passed into another component and used as a callback, can test it.

Reply:
Removed entirely now.